python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -25,6 +25,7 @@
 import copy
 import enum
 import logging
+import time
```

Comment:
Please, don't use the `time` module, you already have timing functionality with the `lsst.ts.utils` module. You can user `utils.current_tai()`.

Reply:
I removed the azEnabled check